src/wp-includes/class-wp-comment-query.php

```diff
@@ -1046,13 +1047,13 @@ protected function fill_descendants( $comments ) {
 			if ( $_parent_ids ) {
 				$cache_keys = array();
 				foreach ( $_parent_ids as $parent_id ) {
-					$cache_keys[ $parent_id ] = "get_comment_child_ids:$parent_id:$key:$last_changed";
+					$cache_keys[ $parent_id ] = "get_comment_child_ids:$parent_id:$key";
 				}
 				$cache_data = wp_cache_get_multiple( array_values( $cache_keys ), 'comment-queries' );
```

Comment:
See my above note on backward compatibility with existing cache keys. My above proposed private function wouldn't work for this, but we could easily adapt it to work with `wp_cache_get_multiple` calls, or have a separate version of it.